keyboards/eason/capsule65/readme.md

```diff
@@ -0,0 +1,13 @@
+# capsule65
+
+A customizable 65% keyboard.
+
+* Keyboard Maintainer: [eason](https://github.com/EasonQian1)
+* Hardware Supported: capsule65
+* Hardware Availability: [eason](https://github.com/EasonQian1)
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

Reply:
It has been changed. Thank you very much